OpenRA.Mods.Common/Traits/Explodes.cs

```diff
@@ -17,11 +18,11 @@ namespace OpenRA.Mods.Common.Traits
 	[Desc("This actor explodes when killed.")]
 	public class ExplodesInfo : ITraitInfo
 	{
-		[WeaponReference, Desc("Weapon to use for explosion if ammo/payload is loaded.")]
+		[WeaponReference, FieldLoader.Require, Desc("Weapon to use for explosion if ammo/payload is loaded.")]
 		public readonly string Weapon = "UnitExplode";
 
 		[WeaponReference, Desc("Weapon to use for explosion if no ammo/payload is loaded.")]
-		public readonly string EmptyWeapon = "UnitExplode";
+		public readonly string EmptyWeapon = null;
```

Comment:
There are several places in our default mods yaml where we don't explicitly give an `EmptyWeapon`.